src/components/ResistanceLogo/ResistanceLogo.js

```diff
@@ -1,27 +1,15 @@
-import React, { PropTypes } from 'react';
+import React from 'react';
 
-const ResistanceLogo = (props) => {
+import styles from './ResistanceLogo.sass';
+
+const ResistanceLogo = () => {
   const image = '../static/img/Logo-large.png';
-  const style = {
-    maxWidth: `${props.width}`,
-    height: `${props.height}`
-  };
 
   return (
-    <div style={style}>
-      <img src={image} style={{ width: '100%' }} alt="Logo for the Resistance Calendar" />
+    <div>
+      <img src={image} className={styles.resistanceLogo} alt="Logo for the Resistance Calendar" />
```

Comment:
Great point. My concern was that using JS to determine CSS styles would mean that we'd have to use JS to detect changes in screen widths and then update the component accordingly if we wanted to change the logo size depending on screen size (ie: iPad user changes from portrait to landscape mode). Any time I can avoid using JS for responsive CSS behavior, I try to do that. 

However, you bring up a good point about making this component more modular, so I went ahead and made it so that you can add a *custom class* to the component, and from there, you can put in as much CSS as you like, including all of the responsive behavior. Whaddyafink?